scene/gui/popup_menu.cpp

```diff
@@ -79,7 +79,8 @@ Size2 PopupMenu::get_minimum_size() const {
 			size.width += check_w + hseparation;
 		}
 
-		String text = items[i].shortcut.is_valid() ? items[i].xl_text + String(tr(items[i].shortcut->get_name())) : items[i].xl_text;
+		String text = items[i].xl_text;
```

Comment:
Are you sure that the width of the potential shortcut string is still properly taken into account after this change?

Reply:
The width of the shortcut is calculated here:
https://github.com/godotengine/godot/blob/d56bc942c194c67c02bc049d8a6518ce0f882324/scene/gui/popup_menu.cpp#L88-L93

It then gets added with the label text width here:
https://github.com/godotengine/godot/blob/d56bc942c194c67c02bc049d8a6518ce0f882324/scene/gui/popup_menu.cpp#L104

But to be 100% sure i wrote the following test:
```GDScript
extends Control

onready var popup : PopupMenu = $MenuButton.get_popup()

var labels = [
	"A short label",
	"A moderately sized label",
	"An extremely long and unnecessarily lengthy label"
]

func _ready():
	popup.connect("index_pressed", self, "_on_index_pressed")

	var sc_short : = ShortCut.new()
	var sc_medium : = ShortCut.new()
	var sc_long : = ShortCut.new()

	var iek_short : = InputEventKey.new()
	var iek_medium : = InputEventKey.new()
	var iek_long : = InputEventKey.new()

	iek_short.set_scancode(KEY_S)

	iek_medium.set_scancode(KEY_M)
	iek_medium.control = true

	iek_long.set_scancode(KEY_L)
	iek_long.alt = true
	iek_long.control = true
	iek_long.meta = true
	iek_long.shift = true

	sc_short.set_shortcut(iek_short)
	sc_medium.set_shortcut(iek_medium)
	sc_long.set_shortcut(iek_long)

	var shortcuts : = [sc_short, sc_medium, sc_long]

	var label_index : = 0
	for i in range(3):
		for j in range(4):
			popup.add_item(labels[i])
		for k in range(4):
			if k == 0:
				label_index += 1
				continue
			popup.set_item_shortcut(label_index, shortcuts[k - 1], true)
			label_index += 1


func _on_index_pressed(index : int) -> void:
	popup.remove_item(index)
```
And here's a gif of the test:
![Gif](https://thumbs.gfycat.com/WhirlwindDisgustingArieltoucan-size_restricted.gif)

Reply:
Awesome work, thanks for the thorough check!